app/src/main/java/com/duckduckgo/app/cta/CtaHelper.kt

```diff
@@ -0,0 +1,95 @@
+/*
+ * Copyright (c) 2020 DuckDuckGo
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.duckduckgo.app.cta
+
+import android.content.Context
+import com.duckduckgo.app.browser.R
+import com.duckduckgo.app.cta.ui.Cta
+import com.duckduckgo.app.cta.ui.DaxBubbleCta
+import com.duckduckgo.app.cta.ui.DaxDialogCta
+import com.duckduckgo.app.cta.ui.HomePanelCta
+import com.duckduckgo.app.global.install.AppInstallStore
+import com.duckduckgo.app.global.install.daysInstalled
+import com.duckduckgo.app.onboarding.store.OnboardingStore
+import com.duckduckgo.app.trackerdetection.model.TrackingEvent
+import javax.inject.Inject
+
+interface CtaHelper {
+    fun addCtaToHistory(newCta: String): String
+
+    fun getTrackersBlockedCtaText(context: Context, trackers: List<TrackingEvent>): String
+
+    fun getNetworkPercentage(network: String): String?
+
+    fun isFromSameNetworkDomain(host: String): Boolean
+
+    fun canSendPixel(cta: Cta): Boolean
+}
+
+class CtaDaxHelper @Inject constructor(val onboardingStore: OnboardingStore, val appInstallStore: AppInstallStore) : CtaHelper {
+
+    override fun canSendPixel(cta: Cta): Boolean {
+        val param = onboardingStore.onboardingDialogJourney?.split("-").orEmpty().toMutableList()
+        return when (cta) {
+            is HomePanelCta -> true
```

Comment:
`HomePanelCta` is not a Dax-CTA, this makes me think about how to be aware of adding a new case here if we create a new Cta in the future, and which ones should be here.

Reply:
This has changed now